src/containers/Dashboard/DashboardLayout.test.jsx

```diff
@@ -68,27 +74,18 @@ describe('DashboardLayout', () => {
         hooks.useDashboardLayoutData.mockReturnValueOnce({ ...hookProps, sidebarShowing: true });
       });
       testColumns();
-      testSnapshot();
-      testSidebarLayout();
+      testSidebarLayout({});
```

Comment:
Passing an empty object here threw me off a bit. I'm wondering how you feel about changing this to something like:
```suggestion
      testSidebarLayout({ isCollapsed: true});
```

And then in the function we wouldn't need the default parameter:
```js
  const testSidebarLayout = ({ isCollapsed }) => {
    it('displays withSidebar width for course list column', () => {
      const courseListCol = screen.getByText('test children').parentElement;
      expect(courseListCol).toHaveClass('col-xl-8');
      const sidebarCol = courseListCol.nextSibling;
      expect(sidebarCol).toHaveClass('sidebar-column', !isCollapsed && 'not-collapsed');
    });
  };
```

This way we are explicit with each function call telling it what we want the variable to be. How do you feel about this change?